pkgs/applications/editors/emacs/elisp-packages/control-lock/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "control-lock";
+
+  src = fetchurl {
+    url = "http://emacswiki.org/emacs/download/control-lock.el";
```

Comment:
This needs to use a version pinned URL.

Reply:
Couldn't find any "versioned" URL.
Seems we do not always require a version (ie: pkgs/applications/editors/emacs/elisp-packages/perl-completion ?)
Or would you be able to suggest a way to create my own ?
Thanks for your help, am new to this !

Reply:
You can use this mirror https://github.com/emacsmirror/emacswiki.org/tree/master

Reply:
And BTW, the pearl-completion derivation should also be fixed.
It just didn't happen yet.

Reply:
Understood -- thank you!